outport/firehose/firehoseIndexerTxPool_test.go

```diff
@@ -0,0 +1,235 @@
+					},
+					{
+						Identifier: []byte("id2"),
+					},
+				},
+			},
+			TxHash: "txHash8",
+		},
+	}
+
+	argsSaveBlock := &outportcore.ArgsSaveBlockData{
+		HeaderHash: headerHashShardV1,
+		Header:     shardHeaderV1,
+		TransactionsPool: &outportcore.Pool{
+			Txs: map[string]data.TransactionHandlerWithGasUsedAndFee{
+				"txHash1": txs[0],
+				"txHash2": txs[1],
+			},
+			Scrs: map[string]data.TransactionHandlerWithGasUsedAndFee{
+				"txHash3": scrs[0],
+			},
+			Rewards: map[string]data.TransactionHandlerWithGasUsedAndFee{
+				"txHash4": rewards[0],
+			},
+
+			Invalid: map[string]data.TransactionHandlerWithGasUsedAndFee{
+				"txHash5": invalidTxs[0],
+			},
+			Receipts: map[string]data.TransactionHandlerWithGasUsedAndFee{
+				"txHash6": receipts[0],
+				"txHash7": receipts[1],
+			},
+			Logs: logs,
+		},
+	}
+
+	firehoseBlock := &firehose.FirehoseBlock{
+		HeaderHash:  headerHashShardV1,
+		HeaderType:  string(core.ShardHeaderV1),
+		HeaderBytes: marshalledHeader,
+		Transactions: map[string]*firehose.TxWithFee{
+			"txHash1": {
+				Transaction: txs[0].TransactionHandler.(*transaction.Transaction),
+				FeeInfo: &firehose.FeeInfo{
+					GasUsed:        txs[0].GasUsed,
+					Fee:            txs[0].Fee,
+					InitialPaidFee: txs[0].InitialPaidFee,
+				},
+			},
+			"txHash2": {
+				Transaction: txs[1].TransactionHandler.(*transaction.Transaction),
+				FeeInfo: &firehose.FeeInfo{
+					GasUsed:        txs[1].GasUsed,
+					Fee:            txs[1].Fee,
+					InitialPaidFee: txs[1].InitialPaidFee,
+				},
+			},
+		},
+		SmartContractResult: map[string]*firehose.SCRWithFee{
+			"txHash3": {
+				SmartContractResult: scrs[0].TransactionHandler.(*smartContractResult.SmartContractResult),
+				FeeInfo: &firehose.FeeInfo{
+					GasUsed:        scrs[0].GasUsed,
+					Fee:            scrs[0].Fee,
+					InitialPaidFee: scrs[0].InitialPaidFee,
+				},
+			},
+		},
+		Rewards: map[string]*rewardTx.RewardTx{
+			"txHash4": rewards[0].TransactionHandler.(*rewardTx.RewardTx),
+		},
+		InvalidTxs: map[string]*firehose.TxWithFee{
+			"txHash5": {
+				Transaction: invalidTxs[0].TransactionHandler.(*transaction.Transaction),
+				FeeInfo: &firehose.FeeInfo{
+					GasUsed:        invalidTxs[0].GasUsed,
+					Fee:            invalidTxs[0].Fee,
+					InitialPaidFee: invalidTxs[0].InitialPaidFee,
+				},
+			},
+		},
+		Receipts: map[string]*receipt.Receipt{
+			"txHash6": receipts[0].TransactionHandler.(*receipt.Receipt),
+			"txHash7": receipts[1].TransactionHandler.(*receipt.Receipt),
+		},
+		Logs: map[string]*transaction.Log{
+			"txHash8": logs[0].LogHandler.(*transaction.Log),
+		},
+	}
+	marshalledFirehoseBlock, err := protoMarshaller.Marshal(firehoseBlock)
+	require.Nil(t, err)
+
+	ioWriterCalledCt := 0
+	ioWriter := &testscommon.IoWriterStub{
+		WriteCalled: func(p []byte) (n int, err error) {
+			ioWriterCalledCt++
+			switch ioWriterCalledCt {
+			case 1:
+				require.Equal(t, []byte("FIRE BLOCK_BEGIN 2\n"), p)
+			case 2:
+
```

Comment:
empty line could be removed

Reply:
Right, removed it from here and other places as well

Reply:
ok